howtos/how-to-add-forwarding/README.md

```diff
@@ -0,0 +1,75 @@
+# How to add message forwarding to your chat
+
+<!-- TO DO: add link to tutorial -->
+
+This is an example project for TalkJS's tutorial on [how to add message forwarding to your chat](ADD_URL). 
+
+This project uses custom [message actions](https://talkjs.com/docs/Features/Message_Features/Message_Actions/) and the [TalkJS REST API](https://talkjs.com/docs/Reference/REST_API/Getting_Started/Introduction/) to allow users to forward text-based messages to conversations that they're part of.
+
+## Prerequisites
+
+To run this example project, you need:
+
+- A [TalkJS account](https://talkjs.com/dashboard/login)
+- [Node.js](https://nodejs.org/en)
+- [npm](https://www.npmjs.com/)
+
+## How to run the example
+
+1. Clone or download the project.
+2. Add a custom message action to allow message forwarding.
+    1. On the **Chat UI** page of your [TalkJS dashboard](https://talkjs.com/dashboard/), in the section 'Custom message actions', select **Add action** to add a message action.
+    2. Give your action the name `forward` and the label `Forward`.
+    3. Select which messages can be forwarded, and who can do the forwarding. For the purposes of this guide, for **Which messages** select that the action should be available for `All messages`, and for **Action available to**, select `Everyone`.
+3. Add styling to distinguish forwarded messages in your theme. The following steps add a subtle but noticeable right-pointed arrow and the label *Forwarded* at the top of the message body of messages that have been forwarded.
+    1. On the **Themes** page of your dashboard, select **Edit** for the theme that you’d like to use.
+    2. To identify user messages that have been forwarded, in `UserMessage`, add the following to the `<template>` part of the code:
+        ```html
+        <!-- Check for forwarded messages -->
+        <div class="{{ custom.forwardedFrom | then: 'forwarded' }}"></div>
+        ```
+    3. Then, to style the message body of a forwarded message, go to `MessageBody` and add the following inside the `<template>` section:
```

Comment:
I put it at the end, is that correct?

Reply:
In this case the location does make a difference, as it'll determine where the label appears. So I've clarified that this should be all the way at the beginning of the template. Thanks so much for bringing this up 🙂